src/containerapp/azext_containerapp/tests/latest/test_containerapp_create_update_with_yaml.py

```diff
@@ -21,18 +22,22 @@ class ContainerappYamlTests(ScenarioTest):
     @AllowLargeResponse(8192)
     @ResourceGroupPreparer(location="westeurope")
     def test_containerapp_preview_create_with_environment_id(self, resource_group):
-        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+        # MSI is not available in North Central US (Stage), if the TEST_LOCATION is "northcentralusstage", use eastus as location
+        location = TEST_LOCATION
+        if format_location(location) == format_location(STAGE_LOCATION):
+            location = "eastus"
+        self.cmd('configure --defaults location={}'.format(location))
 
         env1 = self.create_random_name(prefix='env1', length=24)
         env2 = self.create_random_name(prefix='env2', length=24)
 
         app = self.create_random_name(prefix='yaml1', length=24)
 
-        create_containerapp_env(self, env1, resource_group)
+        create_containerapp_env(self, env1, resource_group, location)
```

Comment:
location is configured in L28. Do we still need to pass it to env creation?

Reply:
Removed.